tests/workflow_test.go

```diff
@@ -23,6 +25,80 @@ const (
 	testNamespace = "default"
 )
 
+func (s *e2eSuite) TestWorkflowExecute_Input() {
+	s.T().Parallel()
+
+	server, cli, _ := s.setUpTestEnvironment()
+	defer func() { _ = server.Stop() }()
+
+	client := server.Client()
+
+	w := s.newWorker(server, testTq, func(r worker.Registry) {
+		r.RegisterWorkflow(encodejson.Workflow)
+	})
+	defer w.Stop()
+
+	// Run the workflow to completion using the CLI.  (TODO: We unfortunately
+	// don't have a way to check the CLI output directly to make sure it prints
+	// the right result...)
+	err := cli.Run([]string{"", "workflow", "execute",
+		"--input", "1", "--input", "\"two\"", "--input", "{\"three\": 3}",
+		"--input", "[\"a\", \"b\", \"c\"]",
```

Comment:
Go tip, use a raw string literal if you need to include a bunch of quotes (https://go.dev/ref/spec#String_literals):

```suggestion
		"--input", `["a", "b", "c"]`,
```

Reply:
Thanks! Just sent you #364 to address these.